python/mlcroissant/recipes/tfds_croissant_builder.ipynb

```diff
@@ -0,0 +1,1540 @@
+      "source": [
+        "To initialize a `CroissantBuilder` in TFDS, we need a Croissant 🥐 file describing a dataset.\n",
+        "\n",
+        "We can use the `from_huggingface_to_croissant` command of the `mlcroissant` library to convert a Hugging Face 🤗 datasets to Croissant JSON-LD files.\n",
+        "\n",
+        "Let's create a Croissant metadata file for [fashion_mnist](https://huggingface.co/datasets/fashion_mnist), a popular dataset for computer vision."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 3,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "vqiO9xo_vqNP",
+        "outputId": "3ba1003a-a4d2-410c-e949-bcf15379d9cc"
+      },
+      "outputs": [
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Downloading builder script: 100% 4.83k/4.83k [00:00<00:00, 17.0MB/s]\n",
+            "Downloading metadata: 100% 3.13k/3.13k [00:00<00:00, 10.9MB/s]\n",
+            "Downloading readme: 100% 8.85k/8.85k [00:00<00:00, 23.6MB/s]\n",
+            "I1127 13:47:02.165981 140028632248320 from_huggingface_to_croissant.py:189] Done. Wrote Croissant JSON-LD to /tmp/croissant_1701092817.7705376.json\n"
+          ]
+        }
+      ],
+      "source": [
+        "!mlcroissant from_huggingface_to_croissant --dataset fashion_mnist"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "YfMQbaK28cAm"
+      },
+      "source": [
+        "## `CroissantBuilder` in TFDS"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "qwTKkcbq-Qz8"
+      },
+      "source": [
+        "Given the Croissant file, we create a TFDS `CroissantBuilder` for the `fashion_mnist` dataset.\n",
+        "\n",
+        "A `CroissantBuilder` takes as input a Croissant 🥐 file, and a list of `RecordSet` names to generate. Each `RecordSet` will correspond to a separated [`BuilderConfig`](https://www.tensorflow.org/datasets/api_docs/python/tfds/core/BuilderConfig)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 5,
+      "metadata": {
+        "colab": {
+          "base_uri": "https://localhost:8080/"
+        },
+        "id": "Nxg8vH-yvlD8",
+        "outputId": "4d7be517-b98d-4fd8-db68-79bef2c5e881"
+      },
+      "outputs": [
+        {
+          "name": "stderr",
+          "output_type": "stream",
+          "text": [
+            "WARNING:absl:Found the following 1 warning(s) during the validation:\n",
+            "  -  [dataset(fashion_mnist)] Property \"https://schema.org/license\" is recommended, but does not exist.\n",
+            "WARNING:absl:Using custom data configuration default\n"
+          ]
+        }
+      ],
+      "source": [
+        "fashion_croissant_path = \"/tmp/croissant_1701092817.7705376.json\"\n",
+        "\n",
+        "fashion_mnist_builder = CroissantBuilder(\n",
```

Comment:
WDYT of:

`fashion_mnist_builder`>`builder`
`fashion_croissant_path`>`file`
etc

To have shorter/easier to read names?